Makefile

```diff
@@ -32,41 +32,40 @@ bin:
 	@mkdir -p $@
 
 .PHONY: driver
-bin/ibm-powervs-block-csi-driver:
 driver: | bin
-	CGO_ENABLED=0 GOOS=linux GOARCH=$(ARCH) go build -ldflags ${LDFLAGS} -o bin/ibm-powervs-block-csi-driver ./cmd/
+	CGO_ENABLED=0 go build -ldflags ${LDFLAGS} -o bin/ibm-powervs-block-csi-driver ./cmd/
 
 .PHONY: test
 test:
 	go test -v -race ./cmd/... ./pkg/...
 
 .PHONY: image
 image:
-	docker build -t $(REGISTRY)/$(IMAGE):$(TAG) . --target debian-base
+	docker build -t $(REGISTRY)/$(IMG) . --target debian-base
```

Comment:
:$(TAG) is missing!

Reply:
`make image` is not for production or pushing to registry.
Used only for testing. Doesn't need a TAG.
However I will add it if required.

Reply:
We already have 
```
ARG TARGETPLATFORM
RUN ARCH=$(echo $TARGETPLATFORM | cut -f2 -d '/') make driver
```
in our docker file.

Do you expect any other changes?

Reply:
we don't need these lines anymore

Reply:
Fixed as expected